app/assets/stylesheets/components/_footer.scss

```diff
@@ -19,7 +19,7 @@
   height: 100%;
   justify-content: space-between;
   padding-bottom: 10px;
-  padding-top: 23px;
```

Comment:
i think perhaps padding-top: 23px should be brought back? I attached screenshoot in the comments, you can see white space between footer and form on 13inch screen, but only very narrow one. You can also see white space on iphone 11 .

Reply:
I have tested it pixel by pixel and 23px is the only value that fitted perfectly. I have tested it in various sizes on my desktop PC and also tested in on my 13" XPS, but I can't recreate the white space effect that you experiencing on your Mac 13.

Reply:
So whatever is the solution to the mentioned issue, it's not this padding.